app/src/main/java/com/islamey/hamza/wazaker/domain/Models/HijriDateResponse.kt

```diff
@@ -0,0 +1,58 @@
+package com.islamey.hamza.wazaker.domain.Models
+
+data class HijriDateResponse(
+    val code: Int,
+    val `data`: HijriDateData,
```

Comment:
why have ` ?

Reply:
as the "data" is a keyword in kotlin and will throw an error, I can change it to any name but I wanted it like the response of api

Reply:
got it, we can name it like `dateData` and use serialized_name as I remember which will use that name for serialization while have another name inside model itself

Reply:
not important, right ?

Reply:
it is nice to know **how** to do it, but not required.